homeassistant/components/hassio/strings.json

```diff
@@ -17,6 +17,23 @@
     }
   },
   "issues": {
+    "issue_addon_detached_addon_missing": {
+      "title": "Missing repository for an installed add-on",
+      "description": "Repository for add-on {addon} is missing. This means it will not get updates and backups may not restore correctly as Supervisor may not be able to build/download the resources required.\n\nPlease check the [add-on's documentation]({addon_url}) for installation instructions and add the repository to the store."
```

Comment:
```suggestion
      "description": "Repository for add-on {addon} is missing. This means it will not get updates, and backups may not be restored correctly as the supervisor may not be able to build/download the resources required.\n\nPlease check the [add-on's documentation]({addon_url}) for installation instructions and add the repository to the store."
```